templates/security/notices/lsn.html

```diff
@@ -0,0 +1,194 @@
+{% extends "security/base_security.html" %}
+
+{% block title %}{{ notice.id }}: {{ notice.title }} | Ubuntu security notices{% endblock %}
+
+{% block content %}
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <div class="col-12">
+	    <h1>{{ notice.id }}: {{ notice.title }}</h1>
+      <p class="p-muted-heading">{{ notice.published.strftime("%d %B %Y") }}</p>
+      <p>{{ notice.summary | safe }}</p>
+      <h2>Releases</h2>
+      <ul class="p-inline-list u-no-margin--bottom">
+        {% for release in notice.releases %}
+          <li class="p-inline-list__item">
+            <a href="/security/notices?release={{ release.codename }}" class="p-button--tag">Ubuntu {{ release.version }} {{ release.support_tag }}</a>
+          </li>
+        {% endfor %}
+      </ul>
+    </div>
+  </div>
+
+  {% if notice.package_descriptions %}
+  <div class="row">
+    <div class="col-12">
+      <h2>Packages</h2>
+      <ul class="p-list">
+        {% for package_description in notice.package_descriptions %}
+          <li class="p-list__item">{{ package_description }}</li>
+        {% endfor %}
+      </ul>
+    </div>
+  {% endif %}
+
+  <div class="row">
+    <div class="col-8">
+      <h2>Details</h2>
+      <p>{{ notice.details|safe }}</p>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-8">
+      <h2>Checking update status</h2>
+      <p>The problem can be corrected in these Livepatch versions:</p>
+      <table>
+        <thead>
+          <th>Kernel type</th>
+          {% for version in notice.instructions %}
+            <th>{{ version }}</th>
+          {% endfor %}
+        </thead>
+        <tbody>
+          {% for instruction_package in notice.instruction_packages %}
+            <tr>
+              <td>{{ instruction_package }}</td>
+              {% for version in notice.instructions %}
+                {% if notice.instructions[version][instruction_package] %}
+                  <td>{{ notice.instructions[version][instruction_package] }}</td>
+                {% else %}
+                  <td>&mdash;</td>
+                {% endif %}
+              {% endfor %}
+            </tr>
+          {% endfor %}
+        </tbody>
+      </table>
+      <p>To check your kernel type and Livepatch version, enter this command:</p>
+      <div class="p-code-copyable">
+        <input class="p-code-copyable__input" value="canonical-livepatch status" readonly="readonly">
+        <button class="p-code-copyable__action">Copy to clipboard</button>
+      </div>
+    </div>
```

Comment:
@sowasred2012 I am trying to add the copyable element on this page. I have copy-pasted the js script at the bottom. Was that right, or was I supposed to import the script neatly?

Reply:
I don't think you need to add or import anything actually - the necessary JS is already present in the project [here](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/static/js/src/scratch.js#L12), so as long as main.js is on the page, you should have it.

Reply:
I investigated a bit, because `main.js` is already present on my lsn template page, and as soon as I delete the block of javascript, the copy does not work anymore. 

I have detected the usage of the `p-code-copyable` on these pages:
https://ubuntu.com/kubernetes/install
https://ubuntu.com/internet-of-things/edgex
https://ubuntu.com/ai/install-kubeflow
https://ubuntu.com/openstack/install

Out of these, the only one working is `/internet-of-things/edgex` because @anthonydillon  also added the script on the page.

`main.js` has some `p-code-copyable` related js inside `/static/scratch.js`, but it seems to be not working.

So I will take the liberty to to replace the code with the script from the docs: https://vanillaframework.io/docs/base/code#copyable
(also I will delete ant's copyable code from `/internet-of-things/edgex`)

As a result all the above links' copyable elements started to work.